pkg/builder3/openapi.go

```diff
@@ -102,7 +104,16 @@ func (o *openAPI) buildOperations(route common.Route, inPathCommonParamsMap map[
 		}
 	}
 
-	// TODO: Default response if needed. Common Response config
+	for code, resp := range o.config.CommonResponses {
+		if _, exists := ret.Responses.StatusCodeResponses[code]; !exists {
+			ret.Responses.StatusCodeResponses[code] = resp
+		}
+	}
+
+	// If there is still no response, use default response provided.
+	if len(ret.Responses.StatusCodeResponses) == 0 {
```

Comment:
hm, I was basing this code off the v2 generator, but you're right we should probably populate it regardless of whether there are other codes.

Reply:
It is not a no-op to populate this field, since its meaning is that if a response code is returned by the API which is not part of `StatusCodeResponses`, the user of the API can assume the response body conforms to the `Default` schema. 

It is important that the spec reflects that actual behavior of the API. In this case, I don't have enough information to know what is the real behavior of the API.  My take is that `ret.Responses.Default` should only ever be non-nil if you can provide that guarantee.

Reply:
Thanks for the explanation, fixed.